### DISCUSSION TITLE
Translation into other languages

### DISCUSSION THREAD

**Discussion by Moonbase59** [javalent/obsidian-leaflet#191] [Ideas]
Now that _Obsidian Leaflet_ somehow secretly has become the "do-all" in mapping, and has gotten much too many options (:grin:) and settings, it just _might_ be time of thinking to make it translatable. I saw a rather good way to do that in @Phibr0’s _Dictionary_ plugin, maybe that can be used?

Or even GNU-gettext-like `.po` files?

_If_ done, it should be done in a way that
- allows you easy integration
- allows foreign helpful translators a standard way of contributing

**Comment by valentine195** [comment]
I would love to implement this. Seems like several plugins are starting to. Will work on it for 4.0.0.

**Comment by phibr0** [comment]
The way I did it is 100% stolen from mgmeyers Kanban Plugin. Should probably add credits for that. (I asked him of course)
